src/Microsoft.Identity.Client/Internal/Cache/AccessTokenCacheKey.cs

```diff
@@ -27,40 +27,25 @@
 
 using System;
 using System.Collections.Generic;
-using System.Globalization;
 using System.Linq;
 using System.Text;
 
 namespace Microsoft.Identity.Client.Internal.Cache
 {
     /// <summary>
-    /// <see cref="TokenCacheKey" /> can be used with Linq to access items from the TokenCache dictionary.
+    /// <see cref="AccessTokenCacheKey" /> can be used with Linq to access items from the TokenCache dictionary.
```

Comment:
A different method name would make it clearer that the string returned has a special property - that it provides a logical identifier that can be used for caching and comparisons.
That isn't normally the case for overrides of _ToString_. It's overridden for a variety of different reasons e.g. to provide a user-friendly description, debug info, or a serialized form of the class. Anyone looking at the code will not immediately expect _ToString_ to return a key value. You could add a doc comment to explain it, but they can get out of date and often people don't read them. Using a different method name can make it immediately obvious without requiring any further explanation.